app/services/session.server.ts

```diff
@@ -75,7 +75,7 @@ export async function createNonceSession({ request, nonce }: { request: Request;
 
 export async function logout(request: Request) {
   const session = await getSession(request);
-  return json(null, {
+  return redirect("/", {
```

Comment:
Hmm, now that I'm thinking about it we might want to redirect to `/app` here instead.

Reply:
Is that not the same thing? Thought it would redirect to the root "/" of the app folder which is /app, but I could be wrong